README.md

```diff
@@ -1,2 +1,175 @@
-# web-file-upload
-A file management system built for vue and react that allows for single and multiple file uploading with a preview feature.
+# Web File Management with Preview - Fully Customized
+
+A versatile and user-friendly file management system built for vue and react that allows for single and multiple file uploading with a preview feature. It allows you to select files and preview them, returning an array of selected files. It allows customizing design by overriding the style classes.
```

Comment:
A versatile and user-friendly file management system built for Vue and React that allows single and multiple file uploading with a preview feature, returning an array of selected files. It supports custom designs by overriding the style classes.